src/main/java/org/jabref/gui/maintable/MainTableTooltip.java

```diff
@@ -23,20 +24,35 @@ public class MainTableTooltip extends Tooltip {
     public MainTableTooltip(DialogService dialogService, GuiPreferences preferences, ThemeManager themeManager, TaskExecutor taskExecutor) {
         this.preferences = preferences;
         this.preview = new PreviewViewer(dialogService, preferences, themeManager, taskExecutor);
-        this.setShowDelay(Duration.seconds(1));
+
+        this.setShowDelay(Duration.millis(100));
+        this.setHideDelay(Duration.millis(500));
+
         this.tooltipContent.getChildren().addAll(fieldValueLabel, preview);
+        tooltipContent.getStyleClass().add("tooltip-content");
```

Comment:
Be consistent with line before

```suggestion
        this.tooltipContent.getStyleClass().add("tooltip-content");
```